package.json

```diff
@@ -17,7 +17,8 @@
     "selectn": "^1.0.20",
     "strong-globalize": "^2.8.0",
     "xml-crypto": "^0.8.4",
-    "xmlbuilder": "^8.2.2"
+    "xmlbuilder": "^8.2.2",
+    "async": "~1.0.0"
```

Comment:
We should use `^2.0.0`. A simple way to add it is `npm i async --save-dev`.

Please make it dev dependency too.